virttest/storage.py

```diff
@@ -555,18 +561,44 @@ def __init__(self, params, root_dir, tag):
                                                               root_dir)
             self.snapshot_format = ss_params.get("image_format")
 
-        self.image_access = retrieve_access_info(self.tag, self.params)
+        self._images_access = retrieve_access_info(self.tag, self.params)
```

Comment:
Instead of simply making it to be a list of the `ImageAccessInfo` objects, we should modify the `ImageAccessInfo` class to make it able to assemble all the individual access info, just like what the `ImageEncryption` class did. The key point is we should distinguish the access info from the current image file's and its backings', then we don't have to implement the lookup method of each caller.